src/components/Main/Header.js

```diff
@@ -0,0 +1,139 @@
+import React, { useState, useEffect } from 'react';
+import Logo from '../../images/logo.png';
+import { NavLink, useNavigate } from 'react-router-dom';
+import useCookie from '../../hooks/useCookie';
+
+function Header () {
+  const navigate = useNavigate();
+
+  const {
+    getCookie,
+    state: { isLoggedIn }
+  } = useCookie();
+
+  const navMenu = [
+    { path: '/', title: 'Home' },
+    { path: '/products', title: 'Shop' },
+    { path: '/contact', title: 'Contact' }
+  ];
+
+  const [isActive, setIsActive] = useState(window.location.pathname);
```

Comment:
There must be useLocation hook for this

Reply:
Not sure that we need here state. You can read url directly, no necessity to call "setIsActive" and perform excessive rerender

Reply:
Fixed.